src/Uno.UWP/UI/Input/GestureRecognizer.cs

```diff
@@ -241,7 +241,9 @@ private static ulong GetPointerIdentifier(PointerPoint point)
 
 			var props = point.Properties;
 
-			ulong identifier = point.PointerId;
+			// Skipping pointer id for touch for now.
+			// It changes on every touch, preventing double tap from working correctly.
+			ulong identifier = point.PointerDeviceType == PointerDeviceType.Touch ? 0 : point.PointerId;
```

Comment:
Not sure if I'm causing any other problems with that.

cc @dr1rrb

Reply:
Yes you will most-probably break multi touch support here. We should fix the pointer ID itself instead ... on which browser do you have this issue?